torchci/components/LogViewer.tsx

```diff
@@ -274,6 +274,9 @@ function LogWithLineSelector({
   });
   // undefined means that no line is selected, so the log viewer is closed
   const [currentLine, setCurrentLine] = useState<number | undefined>(undefined);
+  if (lineNumbers.length === 0) {
+    lineNumbers = [0];
```

Comment:
Maybe a comment on why `[0]` here as I'm trying to understand why not just leave it as an empty array

Reply:
Weirdness wrt the original rockset query.  Its just a hack to make sure the log viewer shows up even when there is no line from log classifier

Reply:
I'm sure future maintainers would find it really helpful to have this described as a comment in the query! :)